api/v1beta1/watcherapi_types.go

```diff
@@ -130,3 +134,18 @@ type WatcherAPIList struct {
 func init() {
 	SchemeBuilder.Register(&WatcherAPI{}, &WatcherAPIList{})
 }
+
+// GetSpecTopologyRef - Returns the LastAppliedTopology Set in the Status
```

Comment:
the function description is wrong, it's the same as the `GetLastAppliedTopology` one

Reply:
Thanks! Fixed.